actions/updatePostLikes.ts

```diff
@@ -1,34 +1,47 @@
 'use server';
 import { auth } from '@/app/api/auth/[...nextauth]/auth';
-import config from '@/config';
-import mumblePostService, {
-  MumblePostService,
-} from '@/services/Mumble/MumblePost';
+
+import mumblePostService from '@/services/Mumble/MumblePost';
 import { revalidatePath } from 'next/cache';
 
-export const increasePostLikes = async (identifier: string): Promise<any> => {
+interface ToggleLikePostArgs {
```

Comment:
Shouldn't there be the letter "I" before the interface name ToggleLikePostArgs?

Reply:
you right. I will update it